src/Auth0.php

```diff
@@ -492,8 +515,10 @@ public function getRefreshToken()
     /**
      * Exchange authorization code for access, ID, and refresh tokens
      *
-     * @throws CoreException - if an active session already or state cannot be validated.
-     * @throws ApiException - if access token is invalid.
+     * @throws CoreException If the state value is missing or invalid.
```

Comment:
This method does **not** throw any new Exceptions, just adding docs for missing ones.